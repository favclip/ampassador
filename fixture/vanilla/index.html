<html>
<head>
    <title>Testing</title>
</head>
<body>
    <h1>Testing</h1>
    <span style="font-style: italic">Italic</span>
</body>
</html>
