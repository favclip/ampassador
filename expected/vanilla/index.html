<!DOCTYPE html><html âš¡>
<head>
    <title>Testing</title>
<style amp-custom>span.h2a-span-2ba262c123{font-style: italic}
</style><link rel="canonical" href="https://example.com/foo/bar"><meta charset="utf-8"><meta content="width=device-width,minimum-scale=1" name="viewport"><style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><script async src="https://cdn.ampproject.org/v0.js"></script><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style><!--from: noscript enclosure for boilerplate--></noscript></head>
<body>
    <h1>Testing</h1>
    <span class="h2a-span-2ba262c123">Italic</span>
</body>
</html>
